stable/README.md

```diff
@@ -1,167 +1,167 @@
 
-| Charts | Privilege | Short Explanation |
-| ----- | ----------- | ------ |
-| transparent-hugepage| allowHostDirVolumePlugin: true | To mount hostPath and disable THP on host|
-| transparent-hugepage| volumes.hostPath | To mount hostPath and disable THP on host|
-| transparent-hugepage| seLinuxContext.* | To mount hostPath and disable THP on host|
-| admin, database| allowedCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
-| admin, database| defaultAddCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
+The results should be as follows:
 
+```bash
+$ helm version
+Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+```
+
+
+# Deploying NuoDB using Helm Charts
+
+The following section outlines the steps in order to deploy a NuoDB database using this Helm Chart repository.
+
+## Configuration Parameters
 
-### Deploy NuoDB (et al) via Helm Charts
+Each Helm Chart has a default values.yaml parameter file that contains configuration parameters specific to that chart. The configuration is structured where configuration values are implemented following a single-definition rule, that is, values are structured and scoped, and shared across charts; e.g. for admin, its parameters are specified once in a single values file which is used for all the charts, and the database chart can use admin values for configuring connectivity of engines to a specific admin process. The same goes for other values **shared** amongst Helm charts. A few key points here:
 
-    IMPORTANT:
-    
-    You MUST first disable THP on nodes where NuoDB will run. Run the `transparent-hugepage` chart first.
+- values files have structure, values are scoped
+- different values files for different deployments
+- values files follow the single definition rule (no repeats)
+- global configuration exists under its own scoped section
+- each chart has its own scoped section named after it
+- cloud information is used to drive availability zones (particularly)
 
-In a nutshell the order of installation is:
+## Deployment Steps
+
+**Note:** You MUST first disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods. Run the `transparent-hugepage` chart first.
 
 - **transparent-hugepage** ([documentation](transparent-hugepage/README.md))
-- **admin** ([documentation](admin/README.md))
-- **database** ([documentation](database/README.md))
 
-See the instructions for the individual charts for deploying the applications.
+Optionally, consider configuring storage classes for persistent storage use. You can still use persistent storage without using the Storage Class Helm Chart and the NuoDB Admin and Database charts will choose a default based on what's available in your Kubernetes  environment.
```

Comment:
Most installations dont have a default and you have to set it. So its not "whats available". There are three options here
- install this chart and select one of the classes in the charts
- install any other classes and select them in the chars
- install any other classes and mark one as the default. No changes to the charts are required then.

Reply:
I've adjusted the text on the stable page to refer to these three options which are now on the storage class chart page.